assets/queries/cloudFormation/sbd_domain_declared_as_a_resource/query.rego

```diff
@@ -0,0 +1,15 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::SDB::Domain"
+  
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":        sprintf("Resources.%s", [name]),
+                "issueType":		   "IncorrectValue",
+                "keyExpectedValue": sprintf("Resources.%s is not defined", [name]),
+                "keyActualValue": 	sprintf("Resources.%s is defined", [name])
+              }
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline